### PR TITLE
move gec demo to new SDK

### PR DESCRIPTION
**Pull request by shaffeeullah**

### CODE REVIEW

File: grammar-error-corrector/backfill.py [hunk 1]
```diff
@@ -4,10 +4,11 @@
 
 from config import logger, GantryConfig, DataStorageConfig
 import gantry
-import gantry.query as gantry_query
+from gantry.query.time_window import TimeWindow
 
 gantry.init(api_key=GantryConfig.GANTRY_API_KEY, send_in_background=False)
-
+# application = gantry.create_application(GantryConfig.GANTRY_APP_NAME)
```
Comment (khluu) [inline grammar-error-corrector/backfill.py:10]: Is it possible to do try/except here to try create_application, if it errors (application is already there), then we call `get_application`?

**Review by khluu — APPROVED** [review]
LGTM!

File: grammar-error-corrector/backfill.py (hunk 1 above)
Comment (khluu) [reply grammar-error-corrector/backfill.py:10]: Then user wouldn't have to comment/uncomment this line while running it again

Comment (shaffeeullah) [reply grammar-error-corrector/backfill.py]: fixed! thank you

**Pull request merged**